### DISCUSSION TITLE
[BUG] Google Reader API configuration test:  FAIL: HTTP error 404 Not Found

### DISCUSSION THREAD

**Discussion by karcio** [FreshRSS/FreshRSS#5757] [Q&A] [answered]
**Describe the bug**
I am trying to use freshrss on mobile. I have turn on Allow API access, and when I check api url I see: HTTP error 404 Not Found

**To Reproduce**
1. Go to 'Settings > Authentication'
2. Click on 'Allow API access'
3. Go to 'Profile' 
4. Click on api url 
5. Verify that endpoint return PASS

**Expected behavior**
User should be able to setup api endpoints for Google Reader

**Screenshots**
![image](https://github.com/FreshRSS/FreshRSS/assets/1783748/9d4611fd-e8d1-4094-b098-d6dcfc490a3b)

**Environment information (please complete the following information):**
- OS: Debian GNU/Linux 11 (bullseye)
- Browser: FF 119.0 
- FreshRSS version: 1.22.0
- Database version: sqlite included in freshrss
- PHP version: 7.4.33
- Installation type: manual
- webserver: nginx 1.18.0 

**Additional context**

**Comment by Alkarex** [comment, marked as answer]
For nginx, make sure you have followed our documentation, in particular the `location` part:
https://freshrss.github.io/FreshRSS/en/admins/10_ServerConfig.html

https://github.com/FreshRSS/FreshRSS/blob/619d3f54a3faaff1b9adf172e090f36faea408c7/docs/en/admins/10_ServerConfig.md?plain=1#L96-L97

**Reply by karcio** [reply]
Hi @Alkarex 
After added this location part now I see: 

```
Google Reader compatible API

Your API address:
    https://digitalhub.duckdns.org/rss/p/api/greader.php
Google Reader API configuration test:
    ❌ FAIL: HTTP error 502 Bad Gateway 
```

**Reply by Alkarex** [reply]
This is a sign of bad nginx configuration. Check the docs again. If you cannot get it to work, other options include Apache, or easier, our ready-to-use Docker image, which does not require so much manual configuration https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md

**Comment by karcio** [comment]
Hi @Alkarex 
Yes, my config had already location for php 
`location ~ \.php$ {`

after comment out this part - all working
Thank you
